presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -67,6 +67,7 @@
     PROCEDURE_CALL_FAILED(0x0001_000F),
     AMBIGUOUS_FUNCTION_IMPLEMENTATION(0x0001_0010),
     ABANDONED_TASK(0x0001_0011),
+    REFRESH_TABLE_FAILED(0x0001_0012),
```

Comment:
This should be a user error. If there are places where it is used as an internal error, create a new error code.